src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -32,37 +61,87 @@
     (range (rand-int 50))))
 
 
+(defn render-import-modal [state props refs]
+  [comps/ModalDialog
+   {:width 600
+    :content (react/create-element
+               [:div {}
+                [:div {:style {:backgroundColor "#fff"
+                               :borderBottom (str "1px solid " (:line-gray style/colors))
+                               :padding "20px 48px 18px"
+                               :fontSize "137%" :fontWeight 400 :lineHeight 1}}
+                 "Import a Method Configuration"]
+                [:div {:style {:padding "22px 48px 40px"
+                               :backgroundColor (:background-gray style/colors)}}
+                 "Destination Name : "
+                 (style/create-text-field
+                   {:defaultValue (get-in @state [:selected-conf "name"])
+                    :ref "destinationName"})
+                 [:br]
+                 "Destination Namespace : "
+                 (style/create-text-field
+                   {:id "render-import-modal-namespace-id-08112015-eas4020c"
+                    :defaultValue (get-in @state [:selected-conf "namespace"])
+                    :ref "destinationNamespace"})
+                 [:br]
+                 [comps/Button
+                  {:title-text "Import configuration"
+                   :icon :plus
+                   :onClick
+                   (fn []
+                     (let [workspace (:workspace props)
+                           dest-name-value (.-value (.getDOMNode (@refs "destinationName")))
+                           dest-namespace-value (.-value (.getDOMNode (@refs "destinationNamespace")))
+                           dest-workspace-name (get workspace "name")
+                           dest-workspace-namespace (get workspace "namespace")
+                           methodRepoQuery (create-import-mc-url-post-data
+                                             (:selected-conf @state)
+                                             dest-name-value
+                                             dest-namespace-value
+                                             dest-workspace-name
+                                             dest-workspace-namespace)
+                           url-method :post
+                           url-path (create-import-mc-url-path
+                                      dest-workspace-namespace
+                                      dest-workspace-name)
+                           url-data (utils/->json-string methodRepoQuery)
+                           url-canned-response {:status 201
+                                                :delay-ms (rand-int 2000)}
+                           url-on-done-func (fn [{:keys [success? xhr]}]
+                                              (if success?
+                                                (utils/rlog "success in import!
```

Comment:
Since you have :refer [rlog] in your imports, you don't need to qualify this with "utils/". Either use `rlog` without the qualifier or remove the :refer stuff above. Your call.